packages/ui/src/components/molecules/Increment/Increment.tsx

```diff
@@ -173,11 +178,23 @@ const Increment = ({
         onPress={handlePlus}
         readonly={readonly}
         disabled={maxValue != null && parseFloat(valueQty) >= maxValue}
+        isBigButton={isBigButton}
+        style={isBigButton && bigButtonStyles.incrementButton}
       />
     </View>
   );
 };
 
+const bigButtonStyles = StyleSheet.create({
+  inputLine: {
+    width: '35%',
+  },
```

Comment:
I don't understand the purpose of this style as you added it to classic style of inputLine. I think you forgot to remove it in classic style